pkgs/applications/misc/gammu/default.nix

```diff
@@ -17,15 +17,23 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-aeaGHVxOMiXRU6RHws+oAnzdO9RY1jw/X/xuGfSt76I=";
   };
 
-  patches = [ ./bashcomp-dir.patch ./systemd.patch ];
+  patches = [
+    ./bashcomp-dir.patch
+    ./systemd.patch
+    (substituteAll {
+      src = ./gammu-config-dialog.patch;
+      dialog = "${dialog}/bin/dialog";
+    })
+  ];
 
   nativeBuildInputs = [ pkg-config cmake ];
 
   strictDeps = true;
 
   buildInputs = [ bluez libusb1 curl gettext sqlite libiconv ]
   ++ optionals dbiSupport [ libdbi libdbiDrivers ]
-  ++ optionals postgresSupport [ postgresql ];
+  ++ optionals postgresSupport [ postgresql ]
+  ++ optionals strictDeps [ bash ];
```

Comment:
strictDeps conditionals is not allowed. Please include this by default

Reply:
Done.